index.bs

```diff
@@ -126,49 +126,176 @@ as Universal Serial Bus (USB), Bluetooth Low Energy (BLE) or Near Field Communic
 
 ## Use Cases ## {#use-cases}
 
-The below use case scenarios illustrate use of two very different types of [=authenticators=], as well as outline further
-scenarios. Additional scenarios, including sample code, are given later in [[#sample-scenarios]].
+The following use case scenarios illustrate a typical flow involving registration and authentication with the sample code 
+included. 
 
-### Registration ### {#usecase-registration}
+### Regirstration ### {#usecase-registration}
 
-- On a phone:
-    * User navigates to example.com in a browser and signs in to an existing account using whatever method they have been using
-        (possibly a legacy method such as a password), or creates a new account.
-    * The phone prompts, "Do you want to register this device with example.com?"
-    * User agrees.
-    * The phone prompts the user for a previously configured [=authorization gesture=] (PIN, biometric, etc.); the user
-        provides this.
-    * Website shows message, "Registration complete."
+This is the first-time flow, in which a new credential is created and registered with the server.
+
+1. The user visits example.com, which serves up a script. At this point, the user must already be logged in using a legacy
+    username and password, or additional authenticator, or other means acceptable to the [=[RP]=].
+
+2. The [=[RP]=] script runs the code snippet below.
+
+3. The client platform searches for and locates the authenticator.
+
+4. The client platform connects to the authenticator, performing any pairing actions if necessary.
```

Comment:
these two steps could be combined?  also, would not pairing occur out-of-band from this flow? i.e., being paired is a pre-condition?